randovania/games/samus_returns/json_data/Surface - West.txt

```diff
@@ -31,7 +31,19 @@ Extra - asset_id: collision_camera_000
   > Save Station
       Trivial
   > Event - Proteus Ridley
-      Trivial
+      All of the following:
+          # TODO: Add fight requirements
+          Any of the following:
+              # Energy Requirements
+              Combat (Advanced) or Normal Damage ≥ 599
+              Combat (Beginner) and Normal Damage ≥ 399
+              Combat (Intermediate) and Normal Damage ≥ 199
+          # Dealing Damage
+          Plasma Beam
+          Any of the following:
+              # Dodging Attacks
+              Space Jump
+              Movement (Advanced) and Use Spider Ball
```

Comment:
would IBJ hypermode also work for dodging?

Reply:
I'm not sure. Sounds likely but never tested.